GameOfLife/Calculations/GameFieldSize.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GameOfLife
+{
+    class GameFieldSize
+    {
+        // Field size set up  2D array
+        public int SelectLenght()
+        {
+            Console.WriteLine("Enter Lenght");
+
+            try
```

Comment:
try-catch are for exceptional cases. They shouldn't be used for data tests. Best approach is to use TryParse method.   
https://docs.microsoft.com/en-us/dotnet/api/system.int32.tryparse?view=net-6.0